### DISCUSSION TITLE
Trouble installing new version

### DISCUSSION THREAD

**Discussion by gtsilva** [BradyAJohnston/MolecularNodes#134] [General]
Hi Brady,
I'm having trouble installing your latest release (2.0.2) on my Blender 3.4.1 (running on Apple M1).
Thoughts?
tks and best,
Guilherme

**Comment by BradyAJohnston** [comment]
Checkout #108 and #111 which should provide a fix for the problem. The M1 distribution of Blender is missing some key files
